.github/workflows/main.yml

```
@@ -0,0 +1,90 @@
+name: Build Linux
+
+on:
+  push:
+    branches:
+      - jellyfin
+
+  pull_request:
+    branches:
+      - jellyfin
+
+  release: 
+    types:
+      - released
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build ${{ matrix.release }} ${{ matrix.arch }}
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04
```
I have no particular opinion toward using Latest or a fixed version but it would prevent GH from 'complaining' that latest will soon switch to 20.04 and you would be using more up to date tools in the background

---

This is about to change this summer so I don't think it's as worth it. Being sticky with latest also reduces future maintenance burden.

---

I know it will change 'soon' :tm:, but it is set to that in the CI I have lately worked on (namely android and kotlin-sdk).
However, I am fine either way, just wanted to mention it :wink: